### PR TITLE
Add more hyperparams for EvolutionOptimizer

### PR DESCRIPTION
**Pull request by UsingtcNower**

### CODE REVIEW

**Review by ljaljushkin — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please address one comment.

File: nncf/config/schemata/defaults.py [hunk 1]
```diff
@@ -59,6 +59,10 @@
 PRUNING_LEGR_TRAIN_STEPS = 200
 PRUNING_LEGR_MAX_PRUNING = 0.8
 PRUNING_LEGR_RANDOM_SEED = 42
+PRUNING_LEGR_POPULATION_SIZE = 64
```
[the remark this answers is not in the document]
Comment (UsingtcNower) [reply nncf/config/schemata/defaults.py:62]: Good catch~ And thanks for your review! I've add it.

**Comment by UsingtcNower** [comment]
> Thanks for the contribution! Please address one comment.

Hi @ljaljushkin ,Thanks for your review! I've addressed the comment, Could you please help to check it?

**Comment by ljaljushkin** [comment]
@UsingtcNower `pre-commit-linters` job is failing because of some issues with code style.
please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style

**Comment by UsingtcNower** [comment]
> @UsingtcNower `pre-commit-linters` job is failing because of some issues with code style. please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style

Hi @ljaljushkin Thanks for your review! I've reformat the code, could you please help to check it? BTW, I found the check tool also pointed out a code outside this change, so I'm not sure whether I need submit it or not.

**Comment by ljaljushkin** [comment]
> > @UsingtcNower `pre-commit-linters` job is failing because of some issues with code style. please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style
> 
> Hi @ljaljushkin Thanks for your review! I've reformat the code, could you please help to check it? BTW, I found the check tool also pointed out a code outside this change, so I'm not sure whether I need submit it or not.

Feel free to submit all diff after `make pre-commit` run. 
on our practice, it affects only modified files.

**Review by ljaljushkin — APPROVED** [review]

**Pull request merged**